shell/platform/tizen/public/flutter_platform_view.h

```diff
@@ -37,35 +39,43 @@ class PlatformView {
                      double dx,
                      double dy) = 0;
   virtual void SetDirection(int direction) = 0;
+
   virtual void ClearFocus() = 0;
-  void SetFocus(bool f) { is_focused_ = f; }
+
+  void SetFocus(bool focused) { is_focused_ = focused; }
+
   bool IsFocused() { return is_focused_; }
 
-  // Key input event
-  virtual void DispatchKeyDownEvent(Ecore_Event_Key* key) = 0;
-  virtual void DispatchKeyUpEvent(Ecore_Event_Key* key) = 0;
+  virtual void DispatchKeyDownEvent(Ecore_Event_Key* event) = 0;
+  virtual void DispatchKeyUpEvent(Ecore_Event_Key* event) = 0;
```

Comment:
Why don't you insert a blank line between method defs? 
I'm just curious about the principle of breaking the line.

Reply:
My personal principle is:
- Separate two members if either (or both) has a block body or comment.
- If two methods do not have comments and have meaningful relationship with each other, a blank line between them might be omitted. Still, it's recommended to document all members whenever possible.